docs/_releases/v2.0.0.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: page
-title: API documentation
+title: Sinon.JS - API documentation
```

Comment:
Yeah, may I change all of the title pages to the better title? I found many titles in documentation is not using good SEO